.travis.yml

```diff
@@ -10,8 +10,8 @@ before_install:
   - travis_retry npm install --loglevel=http
   - >
     openssl aes-256-cbc \
-      -K $encrypted_4fb8e306b739_key \
-      -iv $encrypted_4fb8e306b739_iv \
```

Comment:
Agreed. This was the original approach before I started encrypting the contents. I believe it was `test/config.json`, which is in the `.gitignore`. If it exists, load it. If not and the flag (which should be an environment variable) is set, then throw and exit.